dev-docs/test-3rd-party-apps-manually.md

```diff
@@ -22,43 +22,20 @@ The commands should be available at `https://github.com/GoogleCloudPlatform/ops-
 
 Take Nginx for example:
 ```
-sudo apt update
-sudo apt install -y nginx
+curl https://raw.githubusercontent.com/GoogleCloudPlatform/ops-agent/master/integration_test/third_party_apps_data/applications/nginx/debian_ubuntu/install > install.sh
```

Comment:
[Optional] `install-nginx.sh`?

Reply:
That would require more manual work when copy pasting this command to rename the file when testing one app. I'd defer to the developer whether they want to rename it.

Reply:
The most common case when this instruction is used is when a developer works on one application and test it locally (at least for the past few times I used it). So it's more convenient to copy paste the command, and only need to update one place that has `nginx`.

Either way the developer has the option to rename it as seen fit.

Reply:
Your call.

Reply:
Your call.